### PR TITLE
Weekly Changelog 2023-04-17 to 2023-04-24

### PR DESCRIPTION
**Pull request by kevingranade**
#### Summary
None

#### Additional context
`Automatically generated as a draft. Please copy-edit before merging.`

### CODE REVIEW

File: data/changelog.txt [hunk 1]
```diff
@@ -8,6 +8,10 @@ Added full mouse support to the keybindings screen
 Prevent attacking neutral critter via move in safemode
 Fix and enhance camp radio tower
 Added sorting, showing&hiding pocket contents, and scrolling via mouse in inventory screens
+Add flashlight attachment to hard hats and military helmets
```
Comment (Maleclypse) [inline data/changelog.txt:11]:
```suggestion
```

File: data/changelog.txt [hunk 2]
```diff
@@ -19,6 +23,21 @@ Add several new bungalows
 Updates to several existing maps
 Add the first Exodii mission
 Combat Capable Mounts
+map update: homeless shelter
```
Comment (Maleclypse) [inline data/changelog.txt:26]:
```suggestion
```

File: data/changelog.txt [hunk 3]
```diff
@@ -19,6 +23,21 @@ Add several new bungalows
...
+Three new Scenarios and corresponding starting locations
+Exodii NPC Luliya
+Adds new baby wildlife spawns around the world
```
Comment (Maleclypse) [inline data/changelog.txt:29]:
```suggestion
```

File: data/changelog.txt [hunk 4]
```diff
@@ -19,6 +23,21 @@ Add several new bungalows
 Updates to several existing maps
 Add the first Exodii mission
 Combat Capable Mounts
+map update: homeless shelter
+Three new Scenarios and corresponding starting locations
```
Comment (Maleclypse) [inline data/changelog.txt:27]:
```suggestion
```

File: data/changelog.txt [hunk 5]
```diff
@@ -19,6 +23,21 @@ Add several new bungalows
...
+Exodii NPC Luliya
+Adds new baby wildlife spawns around the world
+Added Winter professions to Brave New World Scenario.
```
Comment (Maleclypse) [inline data/changelog.txt:30]:
```suggestion
```

File: data/changelog.txt [hunk 6]
```diff
@@ -19,6 +23,21 @@ Add several new bungalows
...
+Enables 2x1, 2x2, 3x1 parking lot spawns
+Adds a new profession called Modern Archer that uses the takedown bow and related quiver.
```
Comment (Maleclypse) [inline data/changelog.txt:32]:
```suggestion
```

File: data/changelog.txt [hunk 7]
```diff
@@ -19,6 +23,21 @@ Add several new bungalows
 Updates to several existing maps
 Add the first Exodii mission
 Combat Capable Mounts
+map update: homeless shelter
+Three new Scenarios and corresponding starting locations
+Exodii NPC Luliya
+Adds new baby wildlife spawns around the world
+Added Winter professions to Brave New World Scenario.
+Enables 2x1, 2x2, 3x1 parking lot spawns
```
Comment (Maleclypse) [inline data/changelog.txt:31]:
```suggestion
```

File: data/changelog.txt [hunk 8]
```diff
@@ -19,6 +23,21 @@ Add several new bungalows
...
+Adds a new profession called Modern Archer that uses the takedown bow and related quiver.
+Player can now reload exodii ammunition
```
Comment (Maleclypse) [inline data/changelog.txt:33]:
```suggestion
```

File: data/changelog.txt [hunk 9]
```diff
@@ -19,6 +23,21 @@ Add several new bungalows
...
+Skill level 4 Achievements
+Adds a collection of new snippets, both child notes and E-logs.
```
Comment (Maleclypse) [inline data/changelog.txt:35]:
```suggestion
```

File: data/changelog.txt [hunk 10]
```diff
@@ -19,6 +23,21 @@ Add several new bungalows
...
+Added Proficiency and Weakpoint Set data to zombie jsons
+add new weather: mist and fog. maybe.
+Adds a new profession that starts with a mountable horse: The Cowboy
```
Comment (Maleclypse) [inline data/changelog.txt:38]:
```suggestion
```

File: data/changelog.txt [hunk 11]
```diff
@@ -19,6 +23,21 @@ Add several new bungalows
...
+Add new types of artifacts
+Added a 'Mounted Police Officer' profession, which starts with an armored police horse
```
Comment (Maleclypse) [inline data/changelog.txt:40]:
```suggestion
```

File: data/changelog.txt [hunk 12]
```diff
@@ -28,6 +47,10 @@ Added ability to think to yourself in the message log
 Added Presets for pocket settings
 Prettier loading UI for loading the save
 Add min/max feedback to new character stats interface
+Zebra road markings isolation
```
Comment (Maleclypse) [inline data/changelog.txt:50]:
```suggestion
```

File: data/changelog.txt [hunk 13]
```diff
@@ -28,6 +47,10 @@ Added ability to think to yourself in the message log
...
+Adds a message that details no damage hits against weakpoints with ranged weapons.
+Adds an "ordering" value to each one of the skills inside the skills.json file. It then uses it to set a fixed order to how the skills should be desplayed in the Player Info panel
```
Comment (Maleclypse) [inline data/changelog.txt:52]:
```suggestion
```

File: data/changelog.txt [hunk 14]
```diff
@@ -56,6 +83,13 @@ Cody can make chainmail armor and charges more
 Rework melee, unarmed, dodge, cutting, stabbing, and bashing practice recipes to limit higher level practice actions to books
 Remove flaming eye phantom melee attack that can disrupt aim
 Converts most of the armor and clothing still using the old limb system to the new system with sub-limbs
+partial skill levels contribute to most game tests
+Added small chance to increase follower npc relations during casual chatting
```
Comment (Maleclypse) [inline data/changelog.txt:87]:
```suggestion
```

File: data/changelog.txt [hunk 15]
```diff
@@ -56,6 +83,13 @@ Cody can make chainmail armor and charges more
...
+Adds a plethora of professions to the Crazy Party scenario
```
Comment (Maleclypse) [inline data/changelog.txt:88]:
```suggestion
```

File: data/changelog.txt [hunk 16]
```diff
@@ -56,6 +83,13 @@ Cody can make chainmail armor and charges more
...
+Portal Storm Coherency Pass
+No more domestic tigers
```
Comment (Maleclypse) [inline data/changelog.txt:90]:
```suggestion
```

File: data/changelog.txt [hunk 17]
```diff
@@ -56,6 +83,13 @@ Cody can make chainmail armor and charges more
...
+Adds more possible start locations to the main game
```
Comment (Maleclypse) [inline data/changelog.txt:91]:
```suggestion
```

File: data/changelog.txt [hunk 18]
```diff
@@ -56,6 +83,13 @@ Cody can make chainmail armor and charges more
...
+Adjust the drop rates for bugout bags and basic zombies
```
Comment (Maleclypse) [inline data/changelog.txt:92]:
```suggestion
```

**Review by Maleclypse — COMMENTED** [review]
My first thoughts

File: data/changelog.txt [hunk 19]
```diff
@@ -19,6 +22,11 @@ Add several new bungalows
 Updates to several existing maps
 Add the first Exodii mission
 Combat Capable Mounts
+Exodii NPC Luliya
+Skill level 4 Achievements
+Added Proficiency and Weakpoint Set data to zombie jsons
+add new weather: mist and fog. maybe.
+Add new types of artifacts
```
Comment (dseguin) [inline data/changelog.txt:29]:
```suggestion
Add new Exodii NPC: Luliya
Add new weather: mist and fog. maybe.
```

File: data/changelog.txt [hunk 20]
```diff
@@ -28,6 +36,8 @@ Added ability to think to yourself in the message log
 Added Presets for pocket settings
 Prettier loading UI for loading the save
 Add min/max feedback to new character stats interface
+Adds a message that details no damage hits against weakpoints with ranged weapons.
+Mouse thumb button support
```
Comment (dseguin) [inline data/changelog.txt:40]:
```suggestion
Mouse thumb button support
```

File: data/changelog.txt [hunk 21]
```diff
@@ -45,6 +55,10 @@ Add min/max feedback to new character stats interface
 [Bombastic Perks] New mod to earn perks through gameplay
 [DinoMod] Integrated mutation armor
 [DinoMod] Therizinosaurus
+Museum Location for Xedra Evolved
+Expands some scenario professions with relevant magic professions
+Adds several new professions to the Magiclysm mod
+Added resurrecting meat monstrosities to bombastic perks
```
Comment (dseguin) [inline data/changelog.txt:61]:
```suggestion
[Xedra Evolved] Museum Location for Xedra Evolved
[Magiclysm] Adds several new professions to the Magiclysm mod
[Bombastic Perks] Added resurrecting meat monstrosities to bombastic perks
```

File: data/changelog.txt [hunk 22]
```diff
@@ -86,6 +102,13 @@ Fix using unload_everything zone to remove gunmod gets copies of gunmod
 Ferals can use their guns in GG
 Prevent fire damaging unbreakable items
 Fix energy guns(AFS) on NPC
+Auras no longer burn and do not block bites
+Fix body status menu info not reflecting the selected limb
+Stop clairvoyance breaking Hub 01 and other fixes
+Faction camps now distribute calories based on actual calories and not default calories
+Add mutation ordering for hair variants
+Activate stack of delayed_transformable comestibles in one go
+effect_hallu is effect_visuals instead
```
Comment (dseguin) [inline data/changelog.txt:111]:
```suggestion
Faction camps now distribute calories based on actual calories and not default calories
```

**Pull request merged**